src/utils/auth-utils/removeAuthStateFromLocalStorage.jsx

```diff
@@ -0,0 +1,8 @@
+const removeAuthStateFromLocalStorage = () => {
+  localStorage.removeItem("petkart-auth-token");
+  localStorage.removeItem("petkart-auth-user");
+  sessionStorage.removeItem("petkart-auth-token");
+  sessionStorage.removeItem("petkart-auth-user");
```

Comment:
you can also use localStorage.clear() to clear everything at once.

Reply:
Hey bro, thanks for detailed review!

1. Yes I will remove netlify.
2. Cart page will be merged soon.
3. Error is showing temporary there, soon I will move it to timed toast so this won't be a problem.
4. Thanks.

> * UI looks very good. But you seem to be using 2 deployments (vercel and netlify). So, maybe remove access for this repo from either one of them.
> * Cart page is throwing 404. Check routes config.
> * Error does not reset to `""` or `null` when switched to other pages.
> * Nice error handling btw. Liked it.

Reply:
> you can also use localStorage.clear() to clear everything at once.

Yes I actua

> * UI looks very good. But you seem to be using 2 deployments (vercel and netlify). So, maybe remove access for this repo from either one of them.
> * Cart page is throwing 404. Check routes config.
> * Error does not reset to `""` or `null` when switched to other pages.
> * Nice error handling btw. Liked it.

Hey bro, thanks for detailed review!

1. Yes I will remove netlify.
2. Cart page will be merged soon.
3. Error is showing temporary there, soon I will move it to timed toast so this won't be a problem.
4. Thanks.

Reply:
> you can also use localStorage.clear() to clear everything at once.

Yes but I deliberately used separate removal because in case I write something else in Local or Session Storage in the future for some functionality so that won't be affected.